v63004/outref/gtm8923.txt

```diff
@@ -1,19 +1,61 @@
 setenv gtm_chset UTF-8
 unset echo
-# Testing VIEW "[NO]STATSHARE"[:<region-list>]
+# Testing READ * and WRITE * commands on sockets and files of various UTF CHSETs
 # Create a single region DB with region DEFAULT
 
 # Run gtm8923.m to test WRITE * and READ * commands
-READ * test on UTF-16 file:49
```

Comment:
I like the previous output of each test case in one line instead of the current one with 4 or 5 lines per test case. Can you please fix the test so that output format is preserved.